docs/ai/quickstarts/quickstart-visual-studio-ai-templates.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Quickstart - Create a .NET AI app using the AI app template
+description: Create a .NET AI app to chat with custom data using the AI app template extensions and the Microsoft.Extensions.AI libraries
+ms.date: 2/21/2025
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: alexwolfmsft
+ms.author: alexwolf
+zone_pivot_groups: meai-targets
+---
+
+# Create a .NET AI app to chat with custom data using the AI app template extensions
+
+In this quickstart, you learn how to create a .NET AI app to chat with custom data using the .NET AI app template extensions. These extensions provide additional starter app templates for Visual Studio and the .NET CLI. The templates are designed to streamline the  getting started experience for building AI apps with .NET by handling common setup tasks and configurations for you.
+
+## Prerequisites
+
+:::zone target="docs" pivot="azure-openai"
+
+* .NET 9.0 SDK - [Install the .NET 9.0 SDK](https://dotnet.microsoft.com/download)
+* Visual Studio 2022 - [Install Visual Studio 2022](https://code.visualstudio.com/) (optional)
+* Access to an [Azure OpenAI service](/azure/ai-services/openai/how-to/provisioned-get-started) with the following configurations:
+  - A `gpt-4o-mini` model deployed
+  - A `text-embedding-3-small` model deployed
+  - The [Azure AI Developer](/azure/role-based-access-control/built-in-roles/ai-machine-learning#azure-ai-developer) role assigned to the user you used to sign-in to Visual Studio or the Azure CLI with locally
+
+:::zone-end
+
+:::zone target="docs" pivot="openai"
+
+* .NET 9.0 SDK - [Install the .NET 9.0 SDK](https://dotnet.microsoft.com/download)
+* Visual Studio 2022 - [Install Visual Studio 2022](https://code.visualstudio.com/) (optional)
+* Access to an [OpenAI service](https://openai.com/api/) with the following configurations:
+
+:::zone-end
+
+:::zone target="docs" pivot="ollama"
+
+* .NET 9.0 SDK - [Install the .NET 9.0 SDK](https://dotnet.microsoft.com/download)
+* Visual Studio 2022 - [Install Visual Studio 2022](https://code.visualstudio.com/) (optional)
+* Ollama installed locally - [Install Ollama](https://ollama.com/) locally on your device
+
+:::zone-end
+
+## Install the .NET AI app template
+
+The **AI Chat with Custom Data** template is available as a template package through NuGet. Use the `dotnet new` command to install  the package:
+
+```dotnetcli
+dotnet new --install AIChatCustomData
+```
+
+## Create the .NET AI app
+
+After you have installed the new AI app templates, the template is available through the Visual Studio UI or the .NET CLI.
```

Comment:
We'll also support  VS Code with the C# Dev Kit. Trying to figure out some issues there now.